contribs/gmf/src/search/component.js

```diff
@@ -378,6 +392,13 @@ export class SearchController {
    * Called on initialization of the controller.
    */
   $onInit() {
+    const config = this.optionsName || 'gmfSearchOptions';
+
+    /**
+     * @type {import('gmf/options.js').gmfSearchOptions}
+     */
+    this.options = this.injector_.has(config) ? this.injector_.get(config) : null;
```

Comment:
Why not simply having:
```suggestion
if (this.optionsName) {
   this.options = this.injector_.get(this.optionsName);
}
```

Reply:
I guess there is actually no need to use if at all, and just load the config from the name with the injector...